### PR TITLE
Polish README

### PR DESCRIPTION
**Pull request by sobychacko**

### CODE REVIEW

File: spring-cloud-starter-stream-sink-field-value-counter/README.adoc [hunk 1]
```diff
@@ -1,12 +1,22 @@
...
+`content-type: application/octet-stream`
+
+=== Payload
+
+This sinks supports the following payload types out of the box:
+
+* `POJO (Java bean)`
+* `Map`
+* `Tuple`
+* `JSON String`
```
Comment (artembilan) [inline spring-cloud-starter-stream-sink-field-value-counter/README.adoc]: I think we should describe `content-type` header for all these valid payload types.
when it is `String`, the content-type should be `text/plain` or `application/json`.
And seems for me there is a special content type for the `Tuple`.

File: spring-cloud-starter-stream-sink-field-value-counter/README.adoc [hunk 2]
```diff
@@ -61,5 +75,16 @@ $$spring.redis.url$$:: $$Redis url, which will overrule host, port and password
 == Build
 
 ```
-$ mvn clean package
+$ ./mvnw clean install -PgenerateApps
+$ cd apps
+
+You can find the corresponding binder based projects here. You can then cd into one one of the folders and build it:
```
Comment (artembilan) [inline spring-cloud-starter-stream-sink-field-value-counter/README.adoc]: I think we should consider to present this sentence out of the code block.
See my change for the `aggregator`: https://github.com/spring-cloud-stream-app-starters/aggregator/pull/9/files#diff-84c171210225cbda875c9ddc438ce2abR108

**Review by artembilan — CHANGES_REQUESTED** [review]

File: spring-cloud-starter-stream-sink-field-value-counter/README.adoc (hunk 1 above)
Comment (sobychacko) [reply spring-cloud-starter-stream-sink-field-value-counter/README.adoc]: @artembilan String - `text/plain`, POJO/Map - `applicaiton/octet-stream` and what will be the content type when Tuple is used, do you know?

Comment (artembilan) [reply spring-cloud-starter-stream-sink-field-value-counter/README.adoc]: @sobychacko ,

sorry for delay.
Have been able to find it. See `MessageConverterUtils.X_SPRING_TUPLE`. Looks like this is SCSt feature.
I would really prefer to align `Content-Type` header with the real `payload`.
Thanks

File: spring-cloud-starter-stream-sink-field-value-counter/README.adoc (hunk 2 above)
Comment (ericbottard) [reply spring-cloud-starter-stream-sink-field-value-counter/README.adoc]: 👍

File: spring-cloud-starter-stream-sink-field-value-counter/README.adoc (hunk 1 above)
Comment (ericbottard) [reply spring-cloud-starter-stream-sink-field-value-counter/README.adoc]: Indeed, @artembilan 's suggestion seems to make sense. But then, is the two-section `Input` approach (with a `Headers` and `Payload` subsection) really appropriate? I assume this problem is going to arise elsewhere

**Comment by sobychacko** [comment]
merged the PR.

**Pull request closed without merging**